src/configureApplication.tsx

```diff
@@ -0,0 +1,42 @@
+import log from 'loglevel';
+import Raven from 'raven-js';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { fas } from '@fortawesome/free-solid-svg-icons';
+
+export type ClientEnvVars = {
+  NODE_ENV: string;
+  REACT_APP_SENTRY_DSN: string;
+};
+
+type ConfigureApplicationParams = {
+  _Raven?: typeof Raven;
+  _log?: typeof log;
+  env: ClientEnvVars;
+};
+
+const configureApplication = ({
+  _Raven = Raven,
+  _log = log,
+  env,
```

Comment:
This looks like it can get a default, like `env = (process.env as any) as ClientEnvVars`, right?

Reply:
ah correct, I don't know why I did not do that in the first place 🤦‍♂️ . Fixed, thanks!